photon-ml/src/main/scala/com/linkedin/photon/ml/evaluation/LogisticLossEvaluator.scala

```diff
@@ -19,48 +19,49 @@ import org.apache.spark.rdd.RDD
 import com.linkedin.photon.ml.constants.MathConst
 import com.linkedin.photon.ml.util.Utils
 
-
 /**
-  * Evaluator for logistic loss
-  *
-  * @param labelAndOffsetAndWeights a [[RDD]] of (id, (labels, offsets, weights)) pairs
-  * @param defaultScore the default score used to compute the metric
-  * @author xazhang
-  */
+ * Evaluator for logistic loss
+ *
+ * @param labelAndOffsetAndWeights a [[RDD]] of (id, (labels, offsets, weights)) pairs
+ * @param defaultScore the default score used to compute the metric
+ */
 protected[ml] class LogisticLossEvaluator(
     labelAndOffsetAndWeights: RDD[(Long, (Double, Double, Double))],
     defaultScore: Double = 0.0) extends Evaluator {
 
   /**
-    * Evaluate the scores of the model
-    *
-    * @param scores the scores to evaluate
-    * @return score metric value
-    */
+   * Evaluate the scores of the model
+   *
+   * @param scores the scores to evaluate
+   * @return score metric value
+   */
   def evaluate(scores: RDD[(Long, Double)]): Double = {
     val defaultScore = this.defaultScore
 
-    val scoreAndLabelAndWeights = scores.rightOuterJoin(labelAndOffsetAndWeights)
-        .mapValues { case (scoreOption, (label, offset, weight)) =>
-      (scoreOption.getOrElse(defaultScore) + offset, (label, weight))
-    }.values
+    val scoreAndLabelAndWeights = scores
+      .rightOuterJoin(labelAndOffsetAndWeights)
+      .mapValues { case (scoreOption, (label, offset, weight)) =>
+          (scoreOption.getOrElse(defaultScore) + offset, (label, weight))
+        }
+      .values
 
     scoreAndLabelAndWeights.map { case (score, (label, weight)) =>
-      if (label > MathConst.POSITIVE_RESPONSE_THRESHOLD) {
-        weight * Utils.log1pExp(-score)
-      } else {
-        weight * Utils.log1pExp(score)
+        if (label > MathConst.POSITIVE_RESPONSE_THRESHOLD) {
+          weight * Utils.log1pExp(-score)
+        } else {
+          weight * Utils.log1pExp(score)
+        }
```

Comment:
Can we call ```PointwiseLogisticLossFunction.loss``` directly to get the loss function value, instead of re-defining and computing the loss function value here?

The same goes for ```PoissonLossEvaluator```, ```SmoothedHingeLossEvaluator``` and ```SquaredLossEvaluator```.